### ISSUE TITLE
Downgrading compile version from Java 12 to Java 11

### ISSUE THREAD

**Issue by cwensel** [chklauser/sjsonnet-gradle-plugin#1]
If a project is pinned to Java 11 (LTS), this plugin forces the host Java to be 12 or up to compile.

By republishing compile with Java 11, I believe this hoop can be avoided. 

When debugging a gradle build, it's harder to work around.

I'm only suggesting downgrading to 11 since it's LTS and more common.

From gradle:

```
> Could not resolve io.github.chklauser.sjsonnet:sjsonnet-gradle-plugin:0.1.0-rc.1.
        Required by:
            project :clusterless-scenario > io.github.chklauser.sjsonnet:io.github.chklauser.sjsonnet.gradle.plugin:0.1.0-rc.1
         > No matching variant of io.github.chklauser.sjsonnet:sjsonnet-gradle-plugin:0.1.0-rc.1 was found. The consumer was configured to find a library for use during runtime, compatible with Java 11, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.gradle.plugin.api-version' with value '8.0.2' but:
             - Variant 'apiElements' capability io.github.chklauser.sjsonnet:sjsonnet-gradle-plugin:0.1.0-rc.1 declares a library, packaged as a jar, and its dependencies declared externally:
                 - Incompatible because this component declares a component for use during compile-time, compatible with Java 12 and the consumer needed a component for use during runtime, compatible with Java 11
                 - Other compatible attribute:
                     - Doesn't say anything about org.gradle.plugin.api-version (required '8.0.2')
             - Variant 'runtimeElements' capability io.github.chklauser.sjsonnet:sjsonnet-gradle-plugin:0.1.0-rc.1 declares a library for use during runtime, packaged as a jar, and its dependencies declared externally:
                 - Incompatible because this component declares a component, compatible with Java 12 and the consumer needed a component, compatible with Java 11
                 - Other compatible attribute:
                     - Doesn't say anything about org.gradle.plugin.api-version (required '8.0.2')
```

**Comment by chklauser** [comment]
Hi @cwensel,
Happy to see jsonnet get used 😄 

The error message had me scratching my head for a moment: [the CI pipeline is set to only install Java 11](https://github.com/chklauser/sjsonnet-gradle-plugin/blob/5139fd056310436b95611b178da8501d7feb0fad/.github/workflows/Main.yml#L16). But I _might_ have released that one version manually 😅.

I have released a new version from the pipeline: [1.0.0-rc.1](https://plugins.gradle.org/plugin/io.github.chklauser.sjsonnet/1.0.0-rc.1). That one should definitely _not_ have any references to JDK 12. _However_, a lot has happened since then. Specifically: log4shell. I had to upgrade the Gradle-Version used to build the plugin to the latest 7.x (was 6.x before). 

Does that work for your project? If yes, I'll trigger a 1.0.0 version without the `rc`.
